api/src/opentrons/protocol_engine/state/labware.py

```diff
@@ -320,6 +320,21 @@ def get_well_definition(
                 f"{well_name} does not exist in {labware_id}."
             ) from e
 
+    def get_well_size(
+        self, labware_id: str, well_name: Optional[str] = None
+    ) -> Tuple[float, float, float]:
+        """Get a well's size in x, y, z dimensions based on its shape."""
```

Comment:
Tuples can be confusing, so let's be extra careful with documentation here

```suggestion
        """Get a well's size in x, y, z dimensions based on its shape.
        
        Args:
            labware_id: Labware identifier.
            well_name: Name of well in labware.
            
        Returns:
            A tuple of dimensions in x, y, and z. If well is circular,
            the x and y dimensions will both be set to the diameter.
        """
```